src/components/FilterTodos.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <div class="flex justify-center items-center">
+        <div class="flex flex-col space-y-4 w-xs md:flex-row font-semibold md:w-lg justify-between">
+            <div class="flex space-x-3 justify-between">
+                <button v-for="by in sortBy" :key="by.label"
+                    class="border pl-3 pr-3 h-8 rounded-md text-xs md:text-base transition duration-400 ease-in-out"
+                    :class="by.active ? 'bg-emerald-400 text-white' : 'bg-white text-black'"
+                    @click="$emit('onSort', by.label); resetButtons(by.label);">
```

Comment:
always create a wrapper function when on click you have to run multiple functions